x-pack/plugins/apm/common/agent_name.ts

```diff
@@ -42,20 +42,34 @@ export const AGENT_NAMES: AgentName[] = [
   ...OPEN_TELEMETRY_AGENT_NAMES,
 ];
 
+export const JAVA_AGENT_NAMES: AgentName[] = ['java', 'opentelemetry/java'];
+
+export function isJavaAgentName(
+  agentName?: string
+): agentName is 'java' | 'opentelemetry/java' {
+  return JAVA_AGENT_NAMES.includes(agentName! as AgentName);
+}
+
 export const RUM_AGENT_NAMES: AgentName[] = [
   'js-base',
   'rum-js',
   'opentelemetry/webjs',
 ];
 
-export function isJavaAgentName(
-  agentName: string | undefined
-): agentName is 'java' {
-  return agentName === 'java';
-}
-
 export function isRumAgentName(
   agentName?: string
 ): agentName is 'js-base' | 'rum-js' | 'opentelemetry/webjs' {
   return RUM_AGENT_NAMES.includes(agentName! as AgentName);
 }
+
+export function normalizeAgentName(agentName: string | undefined) {
```

Comment:
```suggestion
export function normalizeAgentName(agentName?: AgentName ) {
```

Reply:
As mentioned above: the input is a string.